src/azure-cli/azure/cli/command_modules/eventgrid/commands.py

```diff
@@ -59,10 +136,82 @@ def load_command_table(self, _):
         g.custom_command('list', 'cli_domain_list')
         g.custom_command('create', 'cli_domain_create_or_update')
         g.command('delete', 'delete')
-        g.generic_update_command('update',
-                                 getter_name='get',
-                                 setter_name='update',
-                                 client_factory=domains_factory)
+        g.custom_command('update', 'cli_domain_update')
+
+    with self.command_group('eventgrid topic private-endpoint-connection', private_endpoint_connections_mgmt_util, client_factory=private_endpoint_connections_factory) as g:
+        g.custom_show_command('show', 'cli_topic_private_endpoint_connection_get')
```

Comment:
https://github.com/Azure/azure-cli/blob/dev/doc/private_endpoint_connection_command_guideline.md Please follow this guideline to support private link scenario. If you want, we can keep these commands as well.

Reply:
@MyronFanQiu  This seems to be a new requirement or guidlines? We followed CLI guidelines to support private link couple of months ago for CLI Extension. Please feel free to open an issue for this against me and we will handle it in separate PR We are fine with the current support we added in 2-3 months ago in CLI extension.

Reply:
oh. I have supported event grid in our central commands. https://github.com/Azure/azure-cli/blob/f5ecf9e2aca18c3409689bda5e8d892b97942e15/src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py#L27-L28

Is this PR same with extension? If so I think we don't need to register twice.

Reply:
But since we already supported event grid in the central commands, do we still need `eventgrid topic private-endpoint-connection` in main module?

Reply:
@MyronFanQiu yes.. let us keep it to ensure our CLI Extension user are not broken (even though we are introducing it here for first time).

Reply:
I know it's the first that `eventgrid topic private-endpoint-connection` are introduced in main repo. Since we already supported same functionality under network command group and we also have event grid extension, do we need `eventgrid topic private-endpoint-connection` in main module as well?

Reply:
OK.. removing..